### PR TITLE
feat(scss) new layout

### LINKED ISSUE #985: [SCSS] Add Footer for fullpage form
just like we have layout with nav menu and header, it would be very usefull to have a layout with a footer

### LINKED ISSUE #1177: New Layout system
Adding the new layout system from @vvalentin-lucca work.

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
New layout (in beta).

fix #985 
fix #1177

-----

image

Capture d’écran 2020-12-14 à 16 03 57

Capture d’écran 2020-12-14 à 16 03 54

Capture d’écran 2020-12-14 à 16 03 50

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1190

File: packages/scss/src/components/_layout.scss [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+	}
+}
+
+.layout-aside-content, .layout-navside-content {
+	flex-grow: 1;
+	display: flex;
+	flex-direction: column;
+}
+
+// le contenu de la navside
```
Comment (BertrandPodevin) [inline packages/scss/src/components/_layout.scss:168]: is that French ? I'm not familiar with it :)

File: packages/scss/src/components/_layout.scss [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+	@include media_smaller_than("md") {
+		// si il précédé par un bouton d’activation du menu non activé
+		.layout-navside-toggle[aria-expanded="false"] ~ & {
+			// est masqué
+			display: none;
+		}
+	}
+}
+
+.layout.mod-noNavsideScrollbar {
```
Comment (BertrandPodevin) [inline packages/scss/src/components/_layout.scss:180]: Here you can factorize .layout and its mod : 
.layout{
   &.mod-noNavsideScrollbar{}
   &.mod-noStickyContentHeader{}
   [...]
}

**Review by BertrandPodevin — COMMENTED** [review]
Has we already discussed, i find the negative mods (`mod-no....`) a bit akwards to use. Otherwise : great work !

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1190

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1190

**Comment by vvalentin-lucca** [comment]
> Has we already discussed, i find the negative mods (`mod-no....`) a bit akwards to use. Otherwise : great work !

You're right, I've dealt with it by avoiding double negation.

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1190

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1190

**Review by sanlucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1190

**Pull request merged**